### DISCUSSION TITLE
Which notation should be use in docs when parameters are explained?

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/keda-docs#539] [Q&A] [answered]
We don't have defined the notation that should be use when you are explaining things like parameters in docs. That's why you can find notations like these:
- `Parameter` - Description
- `Parameter`: Description
- `Parameter:` Description

In the same way, in some cases the sentences finish with a dot and in other cases don't. 

I'd like to review all the docs and unify the styles, but maybe the styles should be decided before it.

WDTY?

**Comment by tomkerkhove** [comment, marked as answer]
> - `Parameter` - Description

This is what is used for the majority and would love to use that.

**Reply by JorTurFer** [reply]
Nice, without final dot, right?

**Reply by tomkerkhove** [reply]
I have missed that part, it should have a `.` at the end since it's a sentence

**Reply by tomkerkhove** [reply]
Created https://github.com/kedacore/keda-docs/issues/540

**Reply by JorTurFer** [reply]
Okey, 
I think the same, but we have to consolidate this part also in all scalers xD
I will start working on this during the weekend 
Thanks!
